index.html

```diff
@@ -107,7 +107,7 @@
                 </tr>
               </table>
             </td>
-            <td ng-if="sectionName==='taken'">
+            <td ng-if="sectionName==='taken' || tableName==='mana'">
```

Comment:
We should probably have a flag in the table definition for uses single element, uses multi element, or null, something like that. This looks awkward IMO